src/main/scala/viper/gobra/reporting/DefaultErrorBackTranslator.scala

```diff
@@ -182,6 +179,12 @@ class DefaultErrorBackTranslator(
           case (err, reason) => err dueTo reason
         }
 
+      case _ / InsufficientPermissionToRangeExpressionAnnotation() =>
+        x.reasons.foldLeft(InsufficientPermissionToRangeExpressionError(x.info): VerificationError){ case (err, reason) => err dueTo reason }
```

Comment:
Why do we need an extra annotation and error? What would be the message shown without this new annotation?

Reply:
It would be "invariant may not be established" and it would point to the whole range-loop structure since that invariant is automatically added and does not exist in the original ast. It would be confusing